data/src/main/java/co/nimblehq/compose/crypto/data/service/ApiService.kt

```diff
@@ -16,4 +17,13 @@ interface ApiService {
         @Query("per_page") itemPerPage: Int,
         @Query("page") page: Int
     ): Response<List<CoinItemResponse>>
+
+    @GET("coins/markets")
+    suspend fun getTrends(
+        @Query("vs_currency") currency: String,
+        @Query("price_change_percentage") priceChangePercentage: String,
+        @Query("order") itemOrder: String,
+        @Query("per_page") itemPerPage: Int,
+        @Query("page") page: Int
+    ): Response<List<TrendingItemResponse>>
```

Comment:
it's the same API to `getCoins`, should we switch repo to use the same API with marking `ids` as null or non-null by `get coins` or `trending coins` cases?

Reply:
Hmmm...Not sure if we leave the `ids` is empty/null the API endpoint accept it or not. @Wadeewee Could you please give a try? 😄

Reply:
I think we can @hoangnguyen92dn @luongvo, but I prefer to separate the cases for `getCoins` and `getTrending` So we don't handle any of the parameters that we don't want to send to `API`, Also it's easy to track for developer what is we do for a calling this `API`, that's why I don't reuse from `getCoins` method, What do you think?

Reply:
@hoangnguyen92dn @Wadeewee tested [here](https://nimble-co.slack.com/archives/C03KBRMUU5U/p1662435970517019?thread_ts=1662433674.291929&cid=C03KBRMUU5U), we can.

Reply:
@luongvo @hoangnguyen92dn Do you want to reuse the `getCoins` method to get the `trending` list? 🙏 
- If yes, I think we can close this PR because @luongvo, you can add null optional in your integrate task.
- If not, I will update `PR` after merging this [PR](https://github.com/nimblehq/jetpack-compose-crypto/pull/27).

Reply:
@luongvo May I know, As we don't separate functions in `Repository` for `getCoin` and `getTrending`, why do we need to create `GetTrendingUseCase`? Why not we rename `GetMyCoinsUseCase` to `GetCoinsUseCase` and reuse it 🙏

Reply:
Because we need to store and pass or not pass the `ids` list. We need to separate the func for VM usage for each trending or my coin list fetching. On another side, repository and service is a lower level of implementation, 2 separate use cases could point to a share API with difference params. Actually we could do it in VM layer but it will mess our VM logic.

Reply:
@Wadeewee ☝️

Reply:
@luongvo Another approach could be create a UseCase with 2 different `Input`: GetMyCoins and `GetTrendingCoins` 😄